wallet/wallet.h

```diff
@@ -55,10 +59,31 @@ enum output_status {
 	output_state_any = 255
 };
 
+static inline enum output_status output_status_in_db(enum output_status s)
+{
+	switch (s) {
```

Comment:
The  enum definition has a `/!\` warning sign, but, why not here also?

Although I suppose someone who decides to change the numbering, even if, there exists already `/!\` in the enum declaration, would not be stopped, by a `/!\` here.